html-links/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Links</title>
+</head>
+<body>
+  <h1>Popular Search Engines</h1>
+  <ul>
+    <li><a href="https://www.google.com/">Google</a></li>
+
```

Comment:
make sure to remove the whitespace from between the elements. you don't want there to be extra spaces like that.

Reply:
Absolutely 👍